.nvmrc

```diff
@@ -1 +1 @@
-lts/hydrogen
\ No newline at end of file
+v22.9.0
```

Comment:
Why not `lts/jod`?

Reply:
Because the types for lts/jod has not released yet, I have checked the npm website.

Reply:
@Amal221200 bro v22 is `lts/jod` itself. I didn't get what you meant. Release page for reference: https://nodejs.org/en/about/previous-releases
![image](https://github.com/user-attachments/assets/4126b20f-3a78-4cda-9a28-a78ff8cd311d)

Reply:
I mean the minor version, the latest version they have released the types are for v22.9.0, not for 22.11.0.

Reply:
@Amal221200 are the potential issues that can arise when using `lts/jod`? If there aren't any significant challenges then please change it to `lts/jod`